src/views/template/factory/factory.html

```diff
@@ -28,8 +28,17 @@
         </form>
     </div>
     <div class="panel-footer">
-        <div class="btn-group" role="group">
-            <button id="buy-truck" type="button" class="btn btn-success pull-right margin-top-10">Buy truck</button>
+        <div class="btn-group btn-group-justified" role="group">
+            <div class="btn-group" role="group">
+                <button id="size-truck" type="button" class="btn btn-default">
+                    <span class="text-success">{{#l}}Increase size trucks{{/l}}</span>
+                </button>
+            </div>
+            <div class="btn-group" role="group">
+                <button id="buy-truck" type="button" class="btn btn-default">
+                    <span class="text-success">{{#l}}Buy extra truck{{/l}}</span>
```

Comment:
These lines ("Increase size trucks" and "Buy extra truck") do not appear to be in the translation file. Could you add them there, too?